src/plugins/visualizations/public/wizard/type_selection/type_selection.tsx

```diff
@@ -95,22 +99,48 @@ class TypeSelection extends React.Component<TypeSelectionProps, TypeSelectionSta
     return (
       <React.Fragment>
         <EuiModalHeader>
-          <EuiModalHeaderTitle>
-            <FormattedMessage
-              id="visualizations.newVisWizard.title"
-              defaultMessage="New Visualization"
-            />
-          </EuiModalHeaderTitle>
+          <EuiFlexGroup direction="column" gutterSize="m">
+            <EuiFlexItem>
+              <EuiModalHeaderTitle>
+                <FormattedMessage
+                  id="visualizations.newVisWizard.title"
+                  defaultMessage="New Visualization"
+                />
+              </EuiModalHeaderTitle>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiCallOut
+                title="Introducing the new Visualization Builder"
+                iconType="visBuilder"
+                heading="h2"
+              >
+                <EuiFlexGroup justifyContent="spaceBetween" wrap={true}>
+                  <EuiFlexItem grow={false}>
+                    <FormattedMessage
+                      id="visualize.tryVisBuilderInfoText"
+                      defaultMessage="Create drag-and-drop visualizations using VisBuilder."
+                    />
+                  </EuiFlexItem>
+                  <EuiFlexItem grow={false}>
+                    <EuiButton
+                      onClick={() => {
+                        this.props.application.navigateToApp('vis-builder');
```

Comment:
Can we add a test for this button?

Reply:
+1 Can you add a unit test for the button within your TypeSelection component, you can use Jest and React Testing Library.